ocfweb/tv/templates/tv/tv.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'tv/base.html' %}
+{% load tv_formatting %}
+
+{% block tv-body %}
+{% include 'tv/hours.html' %}
+{% include 'tv/desktop_usage.html' %}
```

Comment:
do these things really need to be split out into partials?

i think there's an unnecessary amount of indirection going on here

also, indentation

Reply:
I split this apart like this for a future PR where we could add a key to `lab_status.yaml` to display a message on the TV if that became necessary, e.g.

```
{% block tv-body %}
  {% if tv_message %}
    {{ tv_message }}
  {% else %}
    {% include 'tv/hours.html' %}
```
etc.